packit/patches.py

```diff
@@ -371,6 +363,7 @@ def process_patches(
                             # format-patch by default generates patches with prefixes a/ and b/
                             # no-prefix means we don't want those: we need create the patch, again
                             # https://github.com/packit/dist-git-to-source-git/issues/85#issuecomment-698827925
+                            path.unlink()
```

Comment:
Just thinking if it is well-known that this means `remove`.

Reply:
This is synonymous to [`os.unlink()`](https://docs.python.org/3/library/os.html#os.unlink) which is to `os.remove()`. `unlink` seems to be the traditional Unix name.

That said, I'm not sure about how well-known it is. It's certainly documented :)

Should I add a comment that this is about deleting the old patch file before being re-generated?

Reply:
The above comment already kinda says that.
Maybe just 'we need create the patch, again' -> 'we need to delete the patch and create it again'

Reply:
Thanks @jpopelka! I've updated the comment based on your suggestion.